src/node.h

```diff
@@ -335,6 +335,7 @@ struct IsolateSettings {
   v8::PromiseRejectCallback promise_reject_callback = nullptr;
   v8::AllowWasmCodeGenerationCallback
       allow_wasm_code_generation_callback = nullptr;
+  v8::WasmStreamingCallback wasm_streaming_callback = nullptr;
```

Comment:
This makes this semver-major because it’s ABI-breaking, can you add a flag to `IsolateSettingsFlags` that ensures that this field is only accessed if the flag is set (and add that flag to the default value for `flags` here)?

Reply:
Is this more or less accurate:  `IsolateSettings` exists for people embedding Node.js in a larger project?

If yes, are we obliged to provide means to override every callback we are setting in V8? Is it fine to leave it out?

Also: if embedding Node.js is actually supported, how about the following:
 1. Does the embedder have access to primordials and internal modules?
 2. V8 APIs (i.e. updating V8 is semver-major)?
 3. Plugging into snapshot hydration infra (ExternalReferenceRegistry)? Do embeders typically ship a custom snapshot?

Need this info to figure out if we have to let the embedder provide a custom WasmStreamingCallback, in a way that doesn't disable the infrastructure built around WasmStreamingCallback (developed in this PR).

Reply:
just adding the flag to not set it is enough. an example of an embedder is electron, which uses chrome, which sets the callback itself, so all they need to do is tell us not to set it.